charms/openfga-k8s/src/charm.py

```diff
@@ -169,8 +171,13 @@ def _on_stop(self, _):
         """Stop OpenFGA."""
         container = self.unit.get_container(WORKLOAD_CONTAINER)
         if container.can_connect():
-            container.stop("openfga")
-        self._ready()
```

Comment:
Why do we remove the 
` self._ready() `

Reply:
Because the ready function just changes the status and sets the message of the app, I've added a line that will set the status to `Waiting` with a message of "service stopped".